files/pt-br/web/manifest/index.html

```diff
@@ -10,7 +10,7 @@
 ---
 <p>O manifesto do aplicativo web fornece informações sobre uma aplicação (como nome, autor, icone, e descrição) em um arquivo de texto. O propósito do manifesto é instalar aplicações web na tela inicial de um aparelho, propiciando aos usuários um acesso mais rápido e uma experiência enriquecida.</p>
 
-<p>Os manifestos dos aplicativos web fazem parte de um acervo de tecnologias de rede chamadas <a href="/pt-PT/docs/Web/Apps/Progressiva">aplicativos web progressivos</a>, sendo estes aplicativos web que podem ser instalados na tela inicial de um aparelho sem obrigar um usuário a passar por uma loja de aplicativos, dispondo também de outros superpoderes como estar disponíveis offline e apresentando aos usuários notificações push quando houverem mudanças de conteúdo do aplicativo.</p>
+<p>Os manifestos dos aplicativos web fazem parte de um acervo de tecnologias de rede chamadas <a href="https://developer.mozilla.org/pt-BR/docs/Web/Progressive_web_apps">aplicativos web progressivos</a>, sendo estes aplicativos web que podem ser instalados na tela inicial de um aparelho sem obrigar um usuário a passar por uma loja de aplicativos, dispondo também de outros superpoderes como estar disponíveis offline e apresentando aos usuários notificações push quando houverem mudanças de conteúdo do aplicativo.</p>
```

Comment:
Muito obrigado pela contribuição!!!

Uma sugestão para os links internos é manter sempre caminhos absolutos, mas não incluir o protocolo e hostname, dessa forma nada precisará ser feito caso hava uma mudança de domínio _(como já existiu quando a MDN usava Wiki)_.

```suggestion
<p>Os manifestos dos aplicativos web fazem parte de um acervo de tecnologias de rede chamadas <a href="/pt-BR/docs/Web/Progressive_web_apps">aplicativos web progressivos</a>, sendo estes aplicativos web que podem ser instalados na tela inicial de um aparelho sem obrigar um usuário a passar por uma loja de aplicativos, dispondo também de outros superpoderes como estar disponíveis offline e apresentando aos usuários notificações push quando houverem mudanças de conteúdo do aplicativo.</p>
```

Reply:
Muito obrigado pela orientação!